test-avocado/lib/guest-cockpit-redhat.sh

```diff
@@ -2,14 +2,15 @@
 
 spec=$1
 
-set -e
```

Comment:
There is some trouble on RHEL7 what caused that in second round it failed and stops works. I'm not sure why. but disabling set -e improved that bad behaviour.